mewarpx/mewarpx/setups_store/diode_setup.py

```diff
@@ -169,17 +168,17 @@ class DiodeRun_V1(object):
     # steps between doing load-balancing. If <1 load balancing will not be done
     LOAD_BALANCE_INTERVALS = 0
 
-    def __init__(self, **kwargs):
+    def __init__(self, GEOM_STR, **kwargs):
+        # the geometry string must be passed in
+        self.geom_str = GEOM_STR
+        self.rz = self.geom_str == 'RZ'
```

Comment:
```suggestion
        self.rz = (self.geom_str == 'RZ')
```
I know these aren't necessary, but I find them helpful for this specific statement.